securedpreferencestore/src/main/java/devliving/online/securedpreferencestore/SecuredPreferenceStore.java

```diff
@@ -75,16 +73,29 @@ synchronized public static SecuredPreferenceStore getSharedInstance() {
      * @throws InvalidKeyException
      * @throws NoSuchProviderException
      */
-    public static void init( Context appContext,
+    public static void init( Context appContext, String fileName, String keyPrefix,
                              RecoveryHandler recoveryHandler ) throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException, UnrecoverableEntryException, InvalidAlgorithmParameterException, NoSuchPaddingException, InvalidKeyException, NoSuchProviderException {
 
         if(mInstance != null){
-            Log.w("SECURED-PREFERENCE", "init called when there already is a non-null instance of the class");
```

Comment:
logs are important for development as currently we don't have unit tests. But we should definitely not print these in production, so may be keep these but add a check to make sure that we're printing only in DEBUG builds.

Reply:
I'll be wrapping all log and printStackTrace statements in a custom logger class. This will by default only allow logging to happen when the BuildConfig.DEBUG flag is set, with the ability for the end-user to force it on, should they find it necessary.

Reply:
Sounds good!